examples/torch/object_detection/README.md

```diff
@@ -21,13 +23,14 @@ pip install -r examples/torch/requirements.txt
 This scenario demonstrates quantization with fine-tuning of SSD300 on VOC dataset.
 
 #### Dataset preparation
-- Download and extract VOC2007 and VOC2012 train/val and test data + devkit from [here](https://pjreddie.com/projects/pascal-voc-dataset-mirror/)
+- Download and extract in one folder VOC2007 and VOC2012 train/val and test data from [here](https://pjreddie.com/projects/pascal-voc-dataset-mirror/)
```

Comment:
This is too vague. Add an text-based illustration of how the directory structure should look like and which path in this structure corresponds to <path_to_dataset>

Reply:
done